File: python/1176.py
def fibonacci (n):
    r=0
    z=1
    for i in range(0,n):
        aux = z
        z +=r
        r = aux
    return r
    
if __name__:
    n = int(input())
    
    for i in range(0, n):
        x = int(input())
        print(f"Fib({x}) = {fibonacci(x)}")
    

File: php/1176.php
<?php
function fibonacci($n){
     $z=1;
     $r=0;

    for ($i=0;$i<$n;$i++){
        $aux = $z;
        $z +=$r;
        $r = $aux;
    }
    return $r;
}


$n = intval(fgets(STDIN));

for($i=0;$i<$n;$i++){
    $x = intval(fgets(STDIN));
    printf("Fib(%d) = %d\n", $x, fibonacci($x));
}
?>